geomesa-spark/geomesa-spark-sql/src/main/scala/org/apache/spark/sql/SQLGeometricOutputFunctions.scala

```diff
@@ -0,0 +1,50 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.apache.spark.sql
+
+import java.io.StringReader
+
+import com.vividsolutions.jts.geom.{Geometry, Point}
+import org.geotools.geojson.geom.GeometryJSON
+import org.locationtech.geomesa.utils.geohash.GeoHash
+import org.locationtech.geomesa.utils.text.WKBUtils
+
+
+object SQLGeometricOutputFunctions {
+  val geomJSON = new GeometryJSON()
+  val ST_AsBinary: Geometry => Array[Byte] = geom => WKBUtils.write(geom)
+  val ST_AsGeoJSON: Geometry => String = geom => geomJSON.toString(geom)
+  val ST_AsLatLonText: Point => String = point => toLatLonString(point)
+  val ST_AsText: Geometry => String = geom => geom.toText
+  val ST_GeoHash: (Geometry, Int) => String = (geom, prec) => GeoHash(geom.getInteriorPoint, prec).hash
+
+  def registerFunctions(sqlContext: SQLContext): Unit = {
+    sqlContext.udf.register("st_asBinary", ST_AsBinary)
+    sqlContext.udf.register("st_asGeoJSON", ST_AsGeoJSON)
+    sqlContext.udf.register("st_asLatLonText", ST_AsLatLonText)
+    sqlContext.udf.register("st_asText", ST_AsText)
+    sqlContext.udf.register("st_geoHash", ST_GeoHash)
+  }
+
+  private def toLatLonString(point: Point): String = {
```

Comment:
Does this format has a name/designation?  For example, is it IEEE or something similar?  If so, might be worth noting it.